package.json

```diff
@@ -10,7 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@deriv/quill-icons": "latest",
```

Comment:
This caused issue, I had to replace it with the exact version of the library unless we won't have new added icons